src/test/ui/rfc-2008-non-exhaustive/reachable-patterns.stderr

```diff
@@ -1,47 +1,47 @@
 warning: some fields are not explicitly listed
-  --> $DIR/reachable-patterns.rs:127:9
+  --> $DIR/reachable-patterns.rs:129:9
    |
 LL |         VariantNonExhaustive::Bar { x, .. } => {}
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ field `y` not listed
    |
 note: the lint level is defined here
-  --> $DIR/reachable-patterns.rs:124:12
+  --> $DIR/reachable-patterns.rs:126:12
    |
 LL |     #[warn(non_exhaustive_omitted_patterns)]
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    = help: ensure that all fields are mentioned explicitly by adding the suggested fields
    = note: the pattern is of type `VariantNonExhaustive` and the `non_exhaustive_omitted_patterns` attribute was found
```

Comment:
Is this lint supposed to say this instead? I'm not sure what it means by "`non_exhaustive_omitted_patterns` attribute" – that's the name of the lint, not an attribute IIUC.

```suggestion
   = note: the pattern is of type `VariantNonExhaustive` and the `non_exhaustive` attribute was found
```

(Note: Don't apply this suggestion; it's just for demonstration purposes.)

Reply:
Because the lint is allowed by default it needs `#[warn(non_exhaustive_omitted_patterns)]` somewhere on the `match` or the above the wildcard pattern. Now that you point it out its completely redundant info since the snippet shows the lint level. 

Maybe something like `note: the pattern is of type 'VariantNonExhaustive' which is 'non_exhaustive'`

Reply:
Hm good point. I don't think it actually helps the user much to mention `non_exhaustive` here, since there's already a message that mentions the lint itself. "note: the pattern is of type `VariantNonExhaustive`" sounds enough

Reply:
Let's do that in a different PR though. I'll note the comment in the tracking issue.